src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/DisasterRecoveryService.cs

```diff
@@ -64,27 +61,48 @@ internal static ConnectionService ConnectionServiceInstance
         /// </summary>
         public void InitializeService(ServiceHost serviceHost)
         {
+            // Get database info
+            serviceHost.SetRequestHandler(BackupConfigInfoRequest.Type, HandleBackupConfigInfoRequest);
+            // Create backup
             serviceHost.SetRequestHandler(BackupRequest.Type, HandleBackupRequest);
         }
 
-        /// <summary>
-        /// Handles a backup request
-        /// </summary>
-        internal static async Task HandleBackupRequest(
-            BackupParams backupParams,
-            RequestContext<BackupResponse> requestContext)
-        {            
+        public static async Task HandleBackupConfigInfoRequest(
+            DefaultDatabaseInfoParams optionsParams,
+            RequestContext<BackupConfigInfoResponse> requestContext)
+        {
+            var response = new BackupConfigInfoResponse();
             ConnectionInfo connInfo;
             DisasterRecoveryService.ConnectionServiceInstance.TryFindConnection(
-                    backupParams.OwnerUri,
+                    optionsParams.OwnerUri,
                     out connInfo);
-            CDataContainer dataContainer;
+
+            if (connInfo != null)
+            {
+                CDataContainer dataContainer = GetDataContainer(connInfo);
+                SqlConnection sqlConn = GetSqlConnection(connInfo);
+                if (sqlConn != null)
+                {
+                    DisasterRecoveryService.Instance.InitializeBackup(dataContainer, sqlConn);
+                    BackupConfigInfo backupConfigInfo = DisasterRecoveryService.Instance.GetDatabaseInfo(sqlConn.Database);
+                    DatabaseInfo defaultDatabaseInfo = AdminService.GetDefaultDatabaseInfo(connInfo);
```

Comment:
>GetDefaultDatabaseInfo [](start = 68, length = 22)

is this going to get the correct options for the DB, or is it getting the default values from model DB?